src/errors/IISdkCoinErrors.sol

```diff
@@ -16,4 +16,19 @@ interface IISdkCoinErrors {
 
     /// @notice Thrown when a requested operation or action is not supported by the contract
     error Unsupported();
+
+    ///////////////// Invariant Testing Errors
```

Comment:
Note that I added this to debug the invariant testing. Can be moved to specific SdkCoinErrorTest file if we prefer to.

Reply:
If these errors are only used in tests, then they need to be moved before this PR can be merged. Test objects should never be mixed with implementation.